deploy/cdk_exec_policy/cdkExecPolicy.yaml

```diff
@@ -196,7 +196,9 @@ Resources:
             Effect: Allow
             Action:
               - 'cloudformation:*'
-            Resource: !Sub 'arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/CDKToolkit/*'
+            Resource:
+              - !Sub 'arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/CDKToolkit/*'
+              - !Sub 'arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/*PivotRoleStackNestedStack*'
```

Comment:
we could make it more generic and restrict by `dataall` prefixed stacks instead of by pivot role stacks. That way we really point to data.all stacks. Wdyt?

Reply:
@TejasRGitHub @noah-paige

Reply:
agreed - think restricting as described above would be best. It would be by env resource prefix and not necessarily `dataall` right?

Reply:
Agreed! Shall I provide another Update on the same PR ? @dlpzx

Reply:
Yes, I re-requested my review again. Wrong button :S

Reply:
Updated with the environment prefix 😄 , @dlpzx , @noah-paige